packages/sdc/build.js

```
@@ -0,0 +1,572 @@
+    }
+  })
+}
+
+function cli() {
+  const scripts = flags.slice(1)
+  console.log(`Running npm commands: ${scripts.join(' && ')}`)
+  scripts.forEach((script, idx) => {
+    const child = spawn('npm', ['run', script])
+    child.stdout.on('data', (data) => {
+      const color = `\x1b[3${Math.min(idx, 3) + 3}m`
+      const sData = data.toString().replaceAll(/[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g, '') // strip ANSI colours
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)